packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs3/built-in-static-methods-esmodules-target/output.js

```diff
@@ -0,0 +1,148 @@
+var _Symbol$split = require("@babel/runtime-corejs3/core-js-stable/symbol/split");
+
+var _Symbol$search = require("@babel/runtime-corejs3/core-js-stable/symbol/search");
+
+var _Symbol$replace = require("@babel/runtime-corejs3/core-js-stable/symbol/replace");
+
+var _Symbol$match = require("@babel/runtime-corejs3/core-js-stable/symbol/match");
+
+var _Symbol$asyncIterator = require("@babel/runtime-corejs3/core-js-stable/symbol/async-iterator");
+
+var _Reflect$set = require("@babel/runtime-corejs3/core-js-stable/reflect/set");
+
+var _valuesInstanceProperty = require("@babel/runtime-corejs3/core-js-stable/instance/values");
+
+var _keysInstanceProperty = require("@babel/runtime-corejs3/core-js-stable/instance/keys");
+
+var _Object$fromEntries = require("@babel/runtime-corejs3/core-js-stable/object/from-entries");
+
+var _entriesInstanceProperty = require("@babel/runtime-corejs3/core-js-stable/instance/entries");
+
+var _Object$assign = require("@babel/runtime-corejs3/core-js-stable/object/assign");
+
+var _Number$parseFloat = require("@babel/runtime-corejs3/core-js-stable/number/parse-float");
+
+var _Math$hypot = require("@babel/runtime-corejs3/core-js-stable/math/hypot");
+
+var _JSON$stringify = require("@babel/runtime-corejs3/core-js-stable/json/stringify");
+
+Array.from;
+Array.isArray;
+Array.of;
+Date.now;
+_JSON$stringify;
```

Comment:
I was originally thrown off by this, but it's because not all ESM browsers follow the well-formed JSON.stringify spec revision. Overhead here is 2kb.